### PR TITLE
Adding DB_PORTNUMBER

### PR DESCRIPTION
**Pull request by thopiekar**
This PR adds the possibility to use a non-default port.
In my case it is useful, because my Synology NAS uses a more recent MariaDB at a different port than 3306.

This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-docker-files/28)

### CODE REVIEW

**Comment by tholu** [comment]
`DB_PORT` would be a better name for the variable imho.

**Comment by thopiekar** [comment]
Yes, but then you would also need to shorten "DB_HOSTNAME" to "DB_HOST". :wink:

**Comment by tholu** [comment]
@thopiekar Good point 👍

**Comment by thopiekar** [comment]
@tholu Do you know how much time it will take until this gets merged? 🤓

**Comment by tholu** [comment]
My PR #26 is still open and unfortunately I'm not a maintainer of this repository. If you can test the changes in my PR, you can probably accelerate it 😉 Your PR is much smaller and should be merged hopefully soon.

**Comment by alex-sl-eng** [comment]
:+1:

**Review by alex-sl-eng — APPROVED** [review]

**Comment by alex-sl-eng** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-docker-files/28)*

**Comment by thopiekar** [comment]
Great! Thank you for merging this. Hope it will be possible to run Zanata on my Synology NAS now :crossed_fingers:

**Comment by thopiekar** [comment]
Hmm, tried to setup Zanata now, but it seems that my changes about DB_PORTNUMBER have not been uploaded to docker hub so far. When trying to start the container on my NAS I still see this message:
`]) - failure description: "WFLYCTL0211: Cannot resolve expression 'jdbc:mysql://${env.DB_HOSTNAME}:3306/${env.DB_SCHEMA}?characterEncoding=UTF-8'"
`
.. which mentions still the hardcoded 3306 port, whereas I use 3307.
Maybe, in addition, @seanf knows how it comes?

**Comment by huangp** [comment]
That's because we haven't got around to push the new image to docker hub yet. @definite normally handles the release

**Comment by tholu** [comment]
@huangp Since everything about this is moving rather slowly (e.g. no further feedback from anybody on #26), I'm leaning towards publishing my own version on Docker hub.

**Comment by alex-sl-eng** [comment]
@tholu apology for the delay as we were caught up with other tasks scheduled during this period of time. We will look into the PR in docker file as soon as we are available.

**Comment by thopiekar** [comment]
And if possible, it would be nice, if you could push a new image based on the current release. This would allow earlier testing possibilities and thus earlier feedback on changes on docker files.

**Comment by definite** [comment]
@tholu The docker image that includes the merged pull requests is now available as 
docker.io/zanata/server:4.4.3-2
It is also tagged as latest.

**Comment by tholu** [comment]
@definite Thank you!

**Pull request merged**
